pkg/solana/contract.go

```diff
@@ -80,8 +80,8 @@ func NewTracker(spec OCR2Spec, client *Client, transmitter TransmissionSigner, l
 		lggr:            lggr,
 		stateLock:       &sync.RWMutex{},
 		ansLock:         &sync.RWMutex{},
-		stateTime:       time.Now(),
-		ansTime:         time.Now(),
+		stateTime:       time.Time{},
+		ansTime:         time.Time{},
```

Comment:
Alternatively, could omit to get zero implicitly.
```suggestion
```

Reply:
ah yeah that's true. good point